runelite-client/src/test/java/net/runelite/client/plugins/timers/TimersPluginTest.java

```diff
@@ -543,123 +544,43 @@ public void testThrallEnd()
 	// endregion
 
 	@Test
-	public void testImbuedHeartBoost()
+	public void testImbuedHeartStart()
 	{
 		when(timersConfig.showImbuedHeart()).thenReturn(true);
-		when(client.getTickCount()).thenReturn(100);
-		StatChanged event;
-
-		final MenuOptionClicked imbuedHeartClick = new MenuOptionClicked();
-		imbuedHeartClick.setMenuOption("Invigorate");
-		imbuedHeartClick.setId(ItemID.IMBUED_HEART);
-		timersPlugin.onMenuOptionClicked(imbuedHeartClick);
-
-		when(client.getTickCount()).thenReturn(101);
-
-		for (int level = 1, i = 0; level <= Experience.MAX_REAL_LEVEL; level++, i++)
-		{
-			event = new StatChanged(Skill.MAGIC, 0, level, heartBoostedLevel(level));
-			timersPlugin.onStatChanged(event);
+		when(client.getVar(any(Varbits.class))).thenReturn(0);
```

Comment:
This is a no-op. Mock return values use the default value of the method's type--for `int`, that is already 0. Remove this entry from all the heart tests.